### PR TITLE
Open all ColumnFamilies when opening DB

### PR DESCRIPTION
**Pull request by earlbread**
This opens all `ColumnFamilies` when opening DB. Please refer to https://github.com/facebook/rocksdb/wiki/Column-Families#reference.

### CODE REVIEW

File: Libplanet.RocksDBStore/RocksDBStore.cs [hunk 1]
```diff
@@ -106,10 +106,16 @@ public class RocksDBStore : BaseStore
...
             _stateDb = RocksDb.Open(_options, Path.Combine(path, StateDbName));
             _stagedTxDb = RocksDb.Open(_options, Path.Combine(path, StagedTxDbName));
-            _chainDb = RocksDb.Open(
-                _options, Path.Combine(path, ChainDbName), new ColumnFamilies());
-            _stateRefDb = RocksDb.Open(
-                _options, Path.Combine(path, StateRefDbName), new ColumnFamilies());
+
+            // When opening a DB in a read-write mode, you need to specify all Column Families that
+            // currently exist in a DB. https://github.com/facebook/rocksdb/wiki/Column-Families
+            var chainDbPath = Path.Combine(path, ChainDbName);
```
[the remark this answers is not in the document]
Comment (earlbread) [reply Libplanet.RocksDBStore/RocksDBStore.cs]: That would be better. I'll change that.

[the remark this answers is not in the document]
Comment (earlbread) [reply Libplanet.RocksDBStore/RocksDBStore.cs]: Added a commit fd780d1.

File: Libplanet.RocksDBStore/RocksDBStore.cs [hunk 2]
```diff
@@ -971,12 +975,45 @@ private ColumnFamilyHandle GetColumnFamily(RocksDb db, Guid? chainId = null)
...
+
+            foreach (string name in listColumnFamilies)
+            {
+                columnFamilies.Add(name, _options);
+            }
+
+            return columnFamilies;
+        }
+
+        private RocksDb RocksDbOpen(
```
Comment (longfin) [inline Libplanet.RocksDBStore/RocksDBStore.cs]: Trivial, but we prefer this style...

```suggestion
        private RocksDb OpenRocksDb(
```

Comment (earlbread) [reply Libplanet.RocksDBStore/RocksDBStore.cs]: Right, changed. 23555ae

**Pull request merged**